app/tests/v1/test_storemanager.py

```diff
@@ -0,0 +1,114 @@
+from flask import jsonify
+
+
+import unittest
+import app, json
+
+from app.api.v1.models import products
+from app import create_app
+from app.tests.v1.models import TestData
+from app.api.v1.models import Users
+class TestStoreApp(unittest.TestCase):
+
+    """Setup the test client"""
+    def setUp(self):
+        test_app = create_app(config_name='testing')
+        test_app.app_context().push()
+        self.app = test_app.test_client() 
+
+        '''Import all the data to be used for users'''
+        self.sample_data = TestData.sample_data
+        self.sample_sales_data = TestData.sample_sales_data
+        self.sample_user = TestData.sample_user
+
+        #register a sample user
+        self.app.post(
+            '/api/v1/users/registration', 
+            data = json.dumps(self.sample_user), 
+            content_type='application/json'
+            )
+
+        #login a sample user
+        self.auth_token = Users.user_login(self, "mesharkz1@gmail.com", "123123")
+
+        #Add a sample product to the list
+        products.append(self.sample_data)
+
+        
+    def test_whether_returns_status_code_on_products_query(self):
+        self.assertEqual(self.app.get('/api/v1/products', headers = dict(Authorization="Bearer " + self.auth_token), content_type='application/json').status_code, 200)
```

Comment:
Other than testing only status code, also test response messages and data. You can implement this in challenge 3

Reply:
Looks good. Reduce the number of open PRs at the same time, avoid too many commit messages in a single PR. You can start challenge 3

Reply:
Grateful